force-app/main/default/lwc/givingSummary/givingSummary.html

```diff
@@ -16,7 +16,10 @@ <h2 class={summaryTitleClass}>
       </template>
       <template if:false={errorMessage}>
         <lightning-layout-item
-          size="4"
+          size="12"
+          small-device-size="12"
```

Comment:
small-device-size="12" and large-device-size="4" can be omitted. Same on the other lightning layouts